jsonschema-core.xml

```diff
@@ -2405,8 +2405,8 @@
                         </t>
                         <t>
                             This keyword produces an annotation value which is an array of
-                            the indexes to which this keyword validates successfully when applying 
```

Comment:
I've noticed that spacing changes in other PRs have been flagged as "needs to go in a separate PR", so I'll point these out. Also one below.

Reply:
It's not a hard-and-fast rule.  It's preferable to at least put them in separate commits, though.

Reply:
I rebased and I think this went away.